### ISSUE TITLE
Health check logs overload

### ISSUE THREAD

**Issue by vishal-biyani** [fission/fission#505]
Since we have added the health check for fission component pods, the logs of all components is filled with lines & lines of:

```
10.8.0.1 - - [22/Feb/2018:10:58:15 +0000] "GET /healthz HTTP/1.1" 200 0
```
This poses 2 challenges:

- It is difficult to debug, look at logs etc. for ex of controller/executor etc. It is also possible to lose out on important information from logs
- Travis logs are now too big in part due to this so you cannot view the build log unless you download the raw file

**Comment by erwinvaneyk** [comment]
+1 This is indeed an annoying issue

**Comment by vishal-biyani** [comment]
Closed by #525
